### ISSUE TITLE
MapboxMap().getSize throws exception

### ISSUE THREAD

**Issue by artemsorochan** [mapbox/mapbox-maps-flutter#130]
This issue is reproduced on iOS only with example project in map_interface.dart when calling MapboxMap().getSize()

Unhandled Exception: PlatformException(0, Not available., null, null)
#0      _MapInterface.getSize (package:mapbox_maps_flutter/src/pigeons/map_interfaces.dart:3035:7)
<asynchronous suspension>

Flutter (Channel stable, 3.7.7)

**Comment by yazeed-mohammad** [comment]
I have faced same issue,
do you have solve it ?

**Comment by artemsorochan** [comment]
> I have faced same issue, do you have solve it ?

No, our map is using full width, and instead we use MediaQuery.of(context).size.width

**Comment by Bazni** [comment]
Hey 👋,
Any news about it? 

It is coming from the package and not our code right?
